docs/remote/containers.md

```diff
@@ -1146,6 +1155,10 @@ Finally, some extensions (like C#) download secondary dependencies from `downloa
 
 VS Code Server runs on a random port inside the container and VS Code itself uses `docker exec` to communicate with it over Docker's configured communication channel.
 
+### Can I use the extension with source code stored in WSL2 on Windows?
```

Comment:
This also works with WSL 1. I actually couldn't get WSL 1 to work without it. Maybe I'm missing something?

Reply:
Interesting - WSL1 can't run docker in it can it?  I thought that was WSL2 only

Reply:
You're right, that's WSL2 only.